onboarding/mentors.md

```diff
@@ -6,23 +6,63 @@ description: How to be an effective mentor of a new hire
+between people to make them reality. One way we work to maintain and grow this culture is through mentorship.
 
-## What makes a good mentor
-The role of a mentor is purposely different from that of a manager or technical lead, and that role shifts to accommodate the needs of a team member. There may be multiple overlapping descriptions - a mentor may be someone who
-- *Facilitates onboarding*: Simply someone who can help a new hire smooth out the bumps of getting onboarded.
-- *Knows how things work*: From a non-judgmental sounding board for [<strike>stupid</strike> questions](#simple-answers-to-common-questions) to someone who can simply help find the best point person or stakeholder for a project.
-- *Facilitates career development*: From apprenticeship to pointing someone in the direction of an article, project or pair where they can learn something new, mentoring pairs can help uncover or set new goals and offer help in meeting them.
+Although mentorship can occur in many forms, currently on the Artsy engineering team our only formal mentorship
+program is for new hires. Although both the company and engineering team have programs that fall under the umbrella
+of _mentorship, career development_ and _education_, this doc will focus on mentorship in the context of onboarding
+to the engineering team.
 
-It's up to the mentoring pair to find what works best for them and to ensure the relationship evolves over time to remain useful.
+## The Onboarding Mentor
 
-## How to mentor
-*Moving from the abstract to the concrete:*
+Each new engineering hire is assigned an _Onboarding mentor_ are by their manager. Onboarding mentors are not
+managers or teachers, but exist to help new engineers - _regardless of past experience_ - become confident,
+comfortable members of the team. An onboarding mentor acts as a new hire's perhaps most consistent peer (and
+**non-manager**) throughout the sprint rotation process. **As an onboarding mentor, you are expected to**
 
-### Your first day
-- Keep tabs on your mentee's schedule. Their first day will be full of paperwork and meetings with People Ops but some good tasks for a mentor:
+- Support and guide your mentee at least through the first 3 months. Expect that this will take a bit of your time,
+  at least 4-8 hours per week.
+- Schedule at least one meeting per week to check in with your mentee
+- Be an available pair when your mentee gets stuck
+- Provide a safe space for sounding other questions and concerns (though _not_ as a replacement for People Ops)
+- Be a source of consistency throughout their onboarding team sprint rotations
+
+### How to be an onboarding mentor
+
+The role of a mentor is purposely different from that of a manager or technical lead, and that role shifts to
+accommodate the needs of a team member. It's up to the mentoring pair to find what works best for them and to
+ensure the relationship evolves over time to remain useful.
+
+#### What to do?
```

Comment:
(non-blocking) I feel like we could have something in here about addressing different incoming skill levels for new engineers. A junior engineer will have a different onboarding experience and needs vs. a more senior engineer who has had multiple jobs and may already be familiar with a lot of our tooling.